Tests/ASTextNodeTests.m

```diff
@@ -67,7 +68,7 @@ - (void)setUp
 {
   [super setUp];
   _textNode = [[ASTextNode alloc] init];
-  
+  _textNodeBucket = [[NSMutableArray alloc] init];
```

Comment:
Let's make this variable local to the new test method, rather than an instance var.